google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -121,7 +122,13 @@ private Subscriber(Builder builder) throws IOException {
             Ints.saturatedCast(ackExpirationPadding.getStandardSeconds()));
     clock = builder.clock.isPresent() ? builder.clock.get() : CurrentMillisClock.getDefaultClock();
 
-    flowController = new FlowController(builder.flowControlSettings);
+    flowController =
+        new FlowController(
+            builder
+                .flowControlSettings
+                .toBuilder()
+                .setLimitExceededBehavior(LimitExceededBehavior.ThrowException)
```

Comment:
Users will likely just create the FlowControlSetting by calling newBuilder() and not worry setting any LimitExceededBehavior, which will default to whatever the builder sets it to which right now it is the "right value" but if we later change it then the user will start seeing a warning they don't care about. I prefer to not have such log statement.